components/VM_Arm/vm_common.camkes

```diff
@@ -30,9 +30,6 @@ assembly {
         vm.cnode_size_bits = 23;
         vm.simple_untyped24_pool = 12;
 
-        vm.base_prio = 100;
-
-        vm._priority = 101;
```

Comment:
`_priority` has a special meaning and is the default priority for the thread that calls `run`

Reply:
Thanks, worth to add this in a comment. But shouldn't this be 100 then?

Reply:
If the VMM threads aren't a higher priority than the vm thread, the vm won't be preempted when there's work to be done in the VMM which affects things like IRQ latency.

Reply:
But the "VM Thread" is the vCPU thread, right? This is supposed to run at "basePrio - 1".

Reply:
I guess it could still be up to someone to choose to configure the vmm and vcpu threads with the same priority if they wanted to delay things like IRQ injection until the vcpu thread was preempted or yielded, but in general, it's probably most straight forward if the irq server threads have equal or higher prio to vmm thread, which has higher prio to vcpu threads.